src/android/WizPurchase.java

```diff
@@ -462,16 +462,23 @@ private void finishPurchase(JSONArray data) throws JSONException {
 	 * Get the list of purchases
 	 *
 	 **/
-	private JSONArray getPurchases() throws JSONException {
+	private JSONArray restorePurchasesAsPending() throws JSONException {
```

Comment:
I would add `All` in this name to indicate this is not about a selection (function argument) of purchases, but really about all purchases in the system.